docs/source/tutorial.rst

```diff
@@ -568,7 +568,7 @@ As an example, we could submit the operation ``compute_volume`` to the cluster.
 
 ``$ python project.py submit -o compute_volume -n 1 -w 1.5``
 
-This command submits to the cluster the operation ``compute_volume`` for the next available 5 jobs, and each job is submitted with a walltime of 1.5 hours.
+This command submits to the cluster the operation ``compute_volume`` for the next available 1 job, and each job is submitted with a walltime of 1.5 hours.
```

Comment:
Could you rephrase this: "...for the next available job (because we specified -n 1), which is submitted with a walltime of 1.5 hours."?